app/jobs/regular/emit_web_hook_event.rb

```
@@ -0,0 +1,81 @@
+module Jobs
+  class EmitWebHookEvent < Jobs::Base
+    sidekiq_options retry: 5
```

Hmm github basically let it fail and log it. Not sure if we need to retry it